bin/man-n

```diff
@@ -4,6 +4,8 @@
 # Constants.
 #
 
+readonly location="$_" # this must be the first command executed
```

Comment:
nope, you're right. I only tested this locally. When installed through npm (ran it through `npm link`) `man-n` starts failing (e.g. needs to be called as `man-n n`).